File: c#/src/Docmost.Sdk/Models/SearchShareDto.cs
namespace Docmost.Sdk.Models;

public class SearchShareDto
{
    [System.Text.Json.Serialization.JsonPropertyName("query")]
    public string Query { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("shareId")]
    public string ShareId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("spaceId")]
    public string? SpaceId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("limit")]
    public int? Limit { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("offset")]
    public int? Offset { get; set; }

}

File: go/docmost/models/search_share_dto.go
package models

// SearchShareDto from OpenAPI.
type SearchShareDto struct {
	Query string `json:"query"`
	ShareId string `json:"shareId"`
	SpaceId *string `json:"spaceId,omitempty"`
	Limit *int `json:"limit,omitempty"`
	Offset *int `json:"offset,omitempty"`
}
